test/api_callable.js

```diff
@@ -524,3 +525,59 @@ describe('retryable', function() {
     });
   });
 });
+
+describe('bundleable', function() {
+  function byteLength(obj) {
+    return JSON.stringify(obj).length;
+  }
+  function func(argument, metadata, options, callback) {
+    callback(null, argument);
+  }
+  function createRequest(field1, field2) {
+    return {'field1': field1, 'field2': field2};
+  }
+  var bundleOptions = new gax.BundleOptions({'delayThreshold': 10});
```

Comment:
Isn't testing with 'delayThreshold' non-deterministic since we're not mocking the system time?